### ISSUE TITLE
GATK error

### ISSUE THREAD

**Issue by deFerran** [MikkelSchubert/paleomix#26]
Hello,

I'm getting the following error when trying to run Paleomix:

```
Version could not be determined for GenomeAnalysisTK:
 
Attempted to run command:
    $ java -server -Djava.io.tmpdir=/pool/genomics/xxx/xxxxx/xxxxxxx/xxxxxxxxxx/temp -Djava.awt.headless=true -XX:+UseSerialGC -Xmx2g -jar /home/xxxxxxxx/install/jar_root/GenomeAnalysisTK.jar --version
 
Program may be broken or a version not supported by the
pipeline; please refer to the PALEOMIX documentation.
 
    Required:       prior to v4.0.x
    Search string:  ^(?:The Genome Analysis Toolkit \(GATK\) v)?(\d+)\.(\d+)

```
But when I try to run the command showed above:
`$ java -server -Djava.io.tmpdir=/pool/genomics/xxx/xxxxx/xxxxxxx/xxxxxxxxxx/temp -Djava.awt.headless=true -XX:+UseSerialGC -Xmx2g -jar /home/xxxxxxxx/install/jar_root/GenomeAnalysisTK.jar --version`

I get: 3.8-1-0-gf15c1c3ef

So, I`m not sure why it is not working.

Thank you.

**Comment by MikkelSchubert** [comment]
Hi,

The error message that you received should have included additional output
from running the command shown in the error message.
Can you copy/paste that as well?

Best regards,
 Mikkel

**Comment by deFerran** [comment]
Hello Mikkel,
I just found out what the problem was. 
I haven't realized that there was an error log files in "temp" folder. I just check it and it was related to amount of memory available.
As I was just testing to see if I had installed Paleomix right, I was trying to run it with little memory/threads.
I just increased it and now it is working perfectly.
Thank you for your time and for all your work with Paleomix!
Best regards,
Vera
